index.js

```diff
@@ -1861,6 +1861,9 @@ class ForwardEmail {
                 this.config.simpleParser
               );
 
+              // Set header describing original address
+              mail.headers.set("X-ForwardEmail-User", to.address)
```

Comment:
rather common names for a header containing the original envelope to are:

* `X-Envelope-To`
* `X-Original-Recipient`
* `X-Original-To`

Reply:
Yeah, good point. Looking at the descriptions of each header, `X-Original-To` seems like the most sensible one to use. The only problem is (and I'm not too familiar with email headers, so do correct me if I'm mistaken), if the email is forwarded several times, the header might be overriden. I chose something else that isn't used anywhere else so this can't happen.

When I have a bit more time to work on this (in a month or so) I'll find a way to test this properly and see if using `X-Original-To` is reliable.